src/dialogsearch.cpp

```diff
@@ -0,0 +1,179 @@
+#include "dialogsearch.h"
```

Comment:
Could you add the License header to all your new cpp and h files? Simply copy it from some other source file.